doc/user_manual/agents.md

```diff
@@ -0,0 +1,15 @@
+# Agents Overview
+
+Assuming you have your agents registered, you will see them in this list along with lots of useful information:
+
+Act This little check box enabled/disables the agent. Should a Hashcat error occur, the agent will be deactivated automatically unless 'Ignore errors' is enabled for it.
```

Comment:
Ideally, we are consistent in doing such listings, markdown list is missing here